tests/PHPStan/Analyser/data/native-reflection-default-values.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace NativeReflectionDefaultValues;
+
+use function PHPStan\Testing\assertType;
+
+function () {
+	assertType('ArrayObject<*NEVER*, *NEVER*>', new \ArrayObject());
```

Comment:
This isn't ideal but matches the behavior with the actual default value passed (which is the next line)